Resources/Views/Hub/home.leaf

```diff
@@ -268,19 +268,20 @@
                     #endfor
                 </div>
               #endif
-              
-                  #for(session in dropInSessions):
+
+                  #for(session in regularDropInSessions):
                     #extend("Hub/_carousel")
                   #endfor
-                  
+
                   <hr class="mt-3" />
               #endif
               
-              #if(count(groupSessions) > 0):
+              #if(groupDropInSessions):
```

Comment:
Why did we change this? If we ever have a year with zero sessions we'd want to skip this section. It's also not an optional element in the context so it shouldn't be possible to fail this check. 

Recommendation: back to count>0